packages/lit-element-starter-js/rollup.config.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import summary from 'rollup-plugin-summary';
+import {terser} from 'rollup-plugin-terser';
+import resolve from '@rollup/plugin-node-resolve';
+import replace from '@rollup/plugin-replace';
+
+export default {
+  input: 'my-element.js',
+  output: {
+    file: 'my-element.bundled.js',
+    format: 'esm',
+  },
+  onwarn(warning) {
+    if (warning.code !== 'THIS_IS_UNDEFINED') {
+      console.error(`(!) ${warning.message}`);
+    }
+  },
+  plugins: [
+    replace({'Reflect.decorate': 'undefined'}),
+    resolve(),
+    terser({
+      ecma: 2017,
+      module: true,
+      warnings: true,
+      mangle: {
+        properties: {
+          regex: /^__/,
```

Comment:
I think we should add a comment here to highlight this behavior. It could break thing in the docs site if devs access `__` prefixed names somehow? And we might not need it anymore if we ensure the site uses the default build of Lit*

Reply:
It seems reasonable to have this for mangling properties of e.g. the `MyElement` class. But we should probably align this with the single `_` we do in core.... wdyt?